### PR TITLE
LIVE-2147: article video embed

### PR DESCRIPTION
**Pull request by joecowton1**
## Why are you doing this?

Article body video embeds weren't rendering correctly. This PR adds an iframe based `Video` component which takes an `atomId` and `title`.  This can now be used in the `headerMedia` component too. 

## Changes

- Create Editions `Video` component
- Update `MediaAtom` type to include atomId
- add embed url to csp config
- add storybook story

**Storybook doesn't seem to be working at the moment**

## Screenshots

| Before | After |
| --- | --- |
|  |  |

### CODE REVIEW

**Review by abeddow91 — APPROVED** [review]
Nice one!

File: src/components/editions/video/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const Video: FC<Props> = ({ title, atomId }) => {
+	return (
+		<div css={videoWrapperStyles}>
+			<iframe
+				title={title}
+				css={videoStyles}
+				frameBorder="0"
+				scrolling="no"
+				allowFullScreen
+				src={`https://embed.theguardian.com/embed/atom/media/${atomId}#noadsaf`}
```
Comment (JamieB-gu) [inline src/components/editions/video/index.tsx:37]: Question for @paperboyo and @akash1810, is this the correct URL to use for embedding media atoms?

**Review by JamieB-gu — COMMENTED** [review]
Code all looks sensible to me. Just had a question about the correct URL to use for the `iframe` source - dotcom is using `youtube.com` for these.

> Storybook doesn't seem to be working at the moment

Uh oh...

**Comment by faresite** [comment]
> Code all looks sensible to me. Just had a question about the correct URL to use for the `iframe` source - dotcom is using `youtube.com` for these.
> 
> > Storybook doesn't seem to be working at the moment
> 
> Uh oh...

If we used `youtube.com` wouldn't we be losing the benefits of having things like the poster image and the button?

**Comment by JamieB-gu** [comment]
> If we used `youtube.com` wouldn't we be losing the benefits of having things like the poster image and the button?

Don't think so. Dotcom manage that [just fine](https://www.theguardian.com/world/video/2021/apr/15/the-mexican-women-who-kicked-out-the-cartels-video).

It's all [handled by](https://github.com/guardian/atoms-rendering/blob/main/src/YoutubeAtom.tsx) `atoms-rendering`. @nitro-marky may be able to elaborate.

**Pull request merged**